common.go

```diff
@@ -37,7 +39,8 @@ func getMetrics(cmd *cobra.Command, metricsProfile string) []string {
 	case Both:
 		metricsProfiles = []string{"metrics-report.yml", metricsProfile}
 	}
-	return metricsProfiles
+	os.Setenv("TS_METRICS", metricsProfile)
+	os.Setenv("METRICS", strings.Join(metricsProfiles, ","))
```

Comment:
I gather these are different vars for elastic and local indexing ..
I think these allow me to pick a different profile (e.g. aggregated vs non-aggregated) for elastic, but does not allow me to override for the local indexing.

Do we need separate vars here or can we reduce to the same `METRICS` ?

Reply:
> pick a different profile (e.g. aggregated vs non-aggregated)

Looking more, I see we don't get that CLI argument on other workloads, so I won't ask for that as it would be an enhancement.

Also, on the variable consolidation: I see how that allows there to be two different dirs locally for reporting and timeseries metrics .. I don't know that we need separate dirs TBH. I am fine writing to the same dir as it keeps things simpler and doesn't require this new var `TS_METRICS`

Reply:
Tested it locally and the split dir doesn't work anyway. Seems everything went to reporting dir.

```
$ $KUBE_BURNER_OCP version
Version: multiple-indexers
Git Commit: c1c8a2f2b5c4642e3ff48ace4aa7e9173b903794
Build Date: 2024-04-26-11:21:49
Go Version: go1.22.2
OS/Arch: darwin amd64

$ $KUBE_BURNER_OCP cluster-density-v2 --local-indexing --iterations=1
...
time="2024-04-26 11:38:37" level=info msg="Indexing cluster metadata document" file="helpers.go:143"
time="2024-04-26 11:38:37" level=info msg="File reporting-metrics-f85f60bf-8a5f-44bf-87bd-0e5841e2851d/clusterMetadata.json created with 1 documents" file="helpers.go:151"
time="2024-04-26 11:38:37" level=info msg="👋 Exiting kube-burner f85f60bf-8a5f-44bf-87bd-0e5841e2851d" file="helpers.go:111"

$ for i in  *f85f60bf* ; do echo "ls $i | wc" ; ls $i | wc ; done
ls reporting-metrics-f85f60bf-8a5f-44bf-87bd-0e5841e2851d | wc
     103     103    2696
ls ts-metrics-f85f60bf-8a5f-44bf-87bd-0e5841e2851d | wc
       0       0       0
```

Reply:
Yes, there's a bug in the go-commons library, needs to be updated accordingly. We can address it in a future commit I don't consider this an stopper

Reply:
Sounds good to me

Reply:
To be honest its not a bug. Its more of an enhancement that we want to do at `go-commons` level in future. Currently it returns pointer references based on [indexer types](https://github.com/cloud-bulldozer/go-commons/blob/main/indexers/factory.go#L36) that we have defined in past. Meaning if we define multiple indexers with same type `local`, all of them will get overriden with the latest memory address.
```
time="2024-04-26 15:33:00" level=info msg="%!w(map[string]indexers.Indexer=map[indexer-0:0xc00029c9f0 indexer-1:0xc00029c9f0])" file="metrics.go:100"
```
That was the reason we were seeing all the metrics in a single dir. Anyways its non blocking and we can achieve the same with the updated configuration change in this PR. Let me know if you have any further thoughts/concerns.